drivers/i2c/Kconfig.it8xxx2

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2020 ITE Corporation. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+config I2C_ITE_IT8XXX2
+	bool "ITE IT8XXX2 I2C driver"
+	help
+	  Enable I2C support on it8xxx2_evb
```

Comment:
Please expand this to explain

- speeds supported
- repeated start support
- anything else useful?

Reply:
Done